CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.0.0 (2017-05-28)
+
+* Update compatibility to notebook>=5.0
+* Remove kernel activity API in favor of the one in the notebook package
+* Update project overview in the documentation
+* Fix kernel cleanup upon SIGTERM
+* Fix missed environment variables when a kernel is created by a request including
```

Comment:
Does this refer to the whitelist of env variables?
If yes, is the description still correct after PR #248?